### ISSUE TITLE
New Problem: Ridged Loss

### ISSUE THREAD

**Issue by moe18** [Open-Deep-ML/DML-OpenProblem#29]
Implement Ridged regression only using numpy

reference: https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.Ridge.html

**Comment by moe18** [comment]
ill take this one will make it problem (problem # 43)
